src/cljs/org/broadinstitute/firecloud_ui/endpoints.cljs

```diff
@@ -227,6 +227,10 @@
    [{:entityName "foo" :entityType "bar" :succeeded false :message "ohno"}
     {:entityName "bar" :entityType "baz" :succeeded true :message "hooray"}]})
 
+(defn import-attributes [workspace-id]
+      {:path (str "/workspaces/" (ws-path workspace-id) "/importAttributesTSV")
```

Comment:
This indentation looks wrong. Let's figure out how we can get your editor configured correctly.